src/stream/src/executor/managed_state/top_n/top_n_state_new.rs

```diff
@@ -117,15 +117,19 @@ impl<S: StateStore> ManagedTopNStateNew<S> {
         TopNStateRow::new(pk_ordered, row)
     }
 
-    #[cfg(test)]
     /// This function will return the rows in the range of [`offset`, `offset` + `limit`),
     pub async fn find_range(
         &self,
+        pk_prefix: Option<&Row>,
```

Comment:
Please add to the docstring of this function on what's the difference between providing / not providing the pk_prefix.